app/src/main/java/com/example/campusguide/directions/CallbackDirectionsConfirmListener.kt

```diff
@@ -1,9 +1,9 @@
 package com.example.campusguide.directions
 
-class CallbackDirectionsConfirmListener constructor(private val callback: (String, String) -> Unit)
+class CallbackDirectionsConfirmListener constructor(private val callback: (String, String, String) -> Unit)
     : DirectionsDialogConfirmationListener {
 
-    override fun onConfirm(start: String, end: String) {
-        callback(start, end)
+    override fun onConfirm(start: String, end: String, travelMode: String) {
```

Comment:
start, end and travelMode are getting passed around a lot together, maybe it's time to create a class (ex. DirectionsOptions) to hold them together? Would make the code a bit more readable